target/linux/ramips/dts/mt7621_asus_rt-ax53u.dts

```diff
@@ -97,13 +97,13 @@
 
 		partition@7e0000 {
 			label = "ubi";
-			reg = <0x7e0000 0x2e00000>;
+			reg = <0x7e0000 0x7820000>;
 		};
 
-		partition@35e0000 {
-			label = "firmware2";
-			reg = <0x35e0000 0x3200000>;
-		};
+//		partition@35e0000 {
+//			label = "firmware2";
+//			reg = <0x35e0000 0x3200000>;
+//		};
```

Comment:
Please just remove this and not comment it out.

Reply:
Done. Thanks!